### PR TITLE
docs: Add information about the v2 whitelist

### PR DESCRIPTION
**Pull request by maxwellgerber**
the Design Doc for the v2 whitelist feature can be found [here](https://github.com/mulesoft/authentication-server/blob/master/docs/design/connectedApps/Connected_Application_Whitelisting_V2.md)

### CODE REVIEW

File: modules/ROOT/pages/connected-apps-developers.adoc [hunk 1]
```diff
@@ -87,3 +87,11 @@ To create your app, follow these steps:
 . Click *Create App*.
 . Create your app by completing the fields in the *Create App* page.
 . Click *Save*.
+
+== Application Whitelisting
+
+Organization Administrators may enable a whitelist to control which apps can access their users' data.
+When your app is whitelisted, only the set of scopes associated with the app are reviewed and approved by the administrator.
+If you add new scopes to your app at a later date, you will not be able to request those scopes from users until the admin re-approves your app.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-developers.adoc]:
```suggestion
If you add new scopes to your app at a later date, you are unable to request those scopes from users until the admin re-approves your app.
```

File: modules/ROOT/pages/connected-apps-developers.adoc [hunk 2]
```diff
@@ -87,3 +87,11 @@ To create your app, follow these steps:
...
+Existing authorizations and new authorizations using the already-whitelisted set of scopes will continue to work.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-developers.adoc]:
```suggestion
Existing authorizations and new authorizations using the already-whitelisted set of scopes continue to work.
```

File: modules/ROOT/pages/connected-apps-developers.adoc [hunk 3]
```diff
@@ -87,3 +87,11 @@ To create your app, follow these steps:
...
+For more info, see xref:connected-apps-org-admin.adoc#Application-Whitelist[connected apps for admins].
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-developers.adoc]:
```suggestion
For more information, see xref:connected-apps-org-admin.adoc#application-whitelist-admin[connected apps for administrators].
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 4]
```diff
@@ -16,12 +20,30 @@ You can configure one of two types of authorization policies for your users:
 
 * Restricted
 
-  Only connected apps that are approved by an administrator can be authorized by organization members to act on their behalf.
+  Only connected apps that are whitelisted by an administrator can be authorized by organization members to act on their behalf.
+
+By default, the **Unrestricted** policy is enabled. If you change the policy to **Restricted**, the application whitelist will be enabled and all existing application authorizations will be added automatically.
+
+== Application Whitelist
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc:28]:
```suggestion
[[application-whitelist-admin]]
== Application Whitelist
```

File: modules/ROOT/pages/connected-apps-developers.adoc [hunk 5]
```diff
@@ -87,3 +87,11 @@ To create your app, follow these steps:
 . Click *Create App*.
 . Create your app by completing the fields in the *Create App* page.
 . Click *Save*.
+
+== Application Whitelisting
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-developers.adoc]: Whitelist or Whitelisting? Let's keep it consistent, but either one is fine from a docs perspective.

File: modules/ROOT/pages/connected-apps-developers.adoc [hunk 6]
```diff
@@ -87,3 +87,11 @@ To create your app, follow these steps:
 . Click *Create App*.
 . Create your app by completing the fields in the *Create App* page.
 . Click *Save*.
+
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-developers.adoc:90]:
```suggestion

[[application-whitelist-dev]]
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 7]
```diff
@@ -3,10 +3,14 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-As an organization administrator, you are the only user who can view and manage connected apps in Access Management. You can use authorization policies to dictate which apps can be authorized to access user data, and you can whitelist or blacklist apps that users have previously authorized.
-To manage your connected apps, navigate to *Access Management* and select the *Connected Apps* tab. You can add apps in the *Owned Apps* section and manage app access in the *Authorization* section.
+As an organization administrator, you are the only user who can view and manage connected apps in Access Management. You can use authorization policies to dictate which apps can be authorized to access user data, and you can whitelist apps that users would like to use.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
As an organization administrator, you are the only user who can view and manage connected apps in Access Management. You can use authorization policies to dictate which apps can be authorized to access user data, and you can whitelist apps that users want to use.
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 8]
```diff
@@ -3,10 +3,14 @@ ifndef::env-site,env-github[]
...
+To manage your connected apps, navigate to *Access Management* and select the *Connected Apps* tab. You can add apps in the *Owned Apps* section and manage app access in the *Authorizations* section.
+More information about the *Owned Apps* section can be found in the documentation for xref:connected-apps-developers.adoc[app developers].
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
For more information about the *Owned Apps* section, see the documentation for xref:connected-apps-developers.adoc[app developers].
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 9]
```diff
@@ -3,10 +3,14 @@ ifndef::env-site,env-github[]
...
 
-== Authorized Apps
+== Authorizations
+
+You can use this tab to manage which apps are authorized to access your users' data. Click the *...* icon next to an application to view more details, or to remove the app from your organization.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
You can use the *Authorizations* section to manage which apps are authorized to access your users' data. Click the *...* icon next to an application to view more details or to remove the app from your organization.
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 10]
```diff
@@ -16,12 +20,30 @@ You can configure one of two types of authorization policies for your users:
 
 * Restricted
 
-  Only connected apps that are approved by an administrator can be authorized by organization members to act on their behalf.
+  Only connected apps that are whitelisted by an administrator can be authorized by organization members to act on their behalf.
+
+By default, the **Unrestricted** policy is enabled. If you change the policy to **Restricted**, the application whitelist will be enabled and all existing application authorizations will be added automatically.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
By default, the **Unrestricted** policy is enabled. If you change the policy to **Restricted**, the application whitelist is enabled, and all existing application authorizations are added automatically.
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 11]
```diff
@@ -16,12 +20,30 @@ You can configure one of two types of authorization policies for your users:
...
+
+== Application Whitelist
+
+When the whitelist is enabled, only whitelisted applications can be used by organization users. If a user attempts to authorize a non-whitelisted app, they will be prompted to contact their organization administrator.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
When the whitelist is enabled, organization users can use only whitelisted applications. If a user attempts to authorize a non-whitelisted app, they are prompted to contact their organization administrator.
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 12]
```diff
@@ -16,12 +20,30 @@ You can configure one of two types of authorization policies for your users:
...
+
+image::connected-apps-whitelist-blocked.png[The interface the user sees when an app is not on the whitelist, 40%,40%]
+
+Organization administrators will be able to add the app to the whitelist by following the link given to them by the end user.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
Organization administrators are able to add the app to the whitelist by following the link given to them by the end user.
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 13]
```diff
@@ -16,12 +20,30 @@ You can configure one of two types of authorization policies for your users:
...
+
+image::connected-apps-add-to-whitelist.png[The interface the admin sees when they whitelist an app, 40%,40%]
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
image::connected-apps-add-to-whitelist.png[The interface the administrator sees when they whitelist an app, 50%,50%]
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 14]
```diff
@@ -16,12 +20,30 @@ You can configure one of two types of authorization policies for your users:
...
+
+Sometimes, apps will start to ask for additional scopes. This happens when app developers add new features to their products, and need to access more data in your organization.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
Sometimes, apps start to ask for additional scopes. This happens when app developers add new features to their products and need to access more data in your organization.
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 15]
```diff
@@ -16,12 +20,30 @@ You can configure one of two types of authorization policies for your users:
...
+Previously whitelisted apps _must_ be whitelisted again with the new set of scopes before your users may grant those additional scopes to the user.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
Previously whitelisted apps _must_ be whitelisted again with the new set of scopes before your users can grant those additional scopes to the user.
```

Comment (sarathecoubian) [reply modules/ROOT/pages/connected-apps-org-admin.adoc]: Additional scopes to the app?

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 16]
```diff
@@ -16,12 +20,30 @@ You can configure one of two types of authorization policies for your users:
...
 
-You can also manage which apps are authorized to access users' data.
+image::connected-apps-add-to-whitelist-again.png[The interface the admin sees when they whitelist an app again, 40%,40%]
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
image::connected-apps-add-to-whitelist-again.png[The interface the administrator sees when they whitelist an app again, 50%,50%]
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 17]
```diff
@@ -16,12 +20,30 @@ You can configure one of two types of authorization policies for your users:
...
 
-== Whitelist Applications
+Existing authorizations will continue to work if the app is not whitelisted again. New authorizations using the already-whitelisted set of scopes will also be allowed.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
Existing authorizations continue to work if the app is not whitelisted again. New authorizations using the already-whitelisted set of scopes are also allowed.
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 18]
```diff
@@ -16,12 +20,30 @@ You can configure one of two types of authorization policies for your users:
...
+Apps that remove scopes will not need to be whitelisted again.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
Apps that remove scopes do not need to be whitelisted again.
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 19]
```diff
@@ -16,12 +20,30 @@ You can configure one of two types of authorization policies for your users:
...
 
-Using the Connected Apps feature, you can view which users have granted access to apps as well as the specified access scopes. You can revoke or reinstate access to the apps at any time.
+[NOTE]
+--
+Applications created by users in the same organization are marked as *internal apps*, and do not need to be whitelisted manually.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
Applications created by users in the same organization are marked as *internal apps* and do not need to be whitelisted manually.
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc [hunk 20]
```diff
@@ -16,12 +20,30 @@ You can configure one of two types of authorization policies for your users:
...
-  Only connected apps that are approved by an administrator can be authorized by organization members to act on their behalf.
+  Only connected apps that are whitelisted by an administrator can be authorized by organization members to act on their behalf.
+
+By default, the **Unrestricted** policy is enabled. If you change the policy to **Restricted**, the application whitelist will be enabled and all existing application authorizations will be added automatically.
+
+== Application Whitelist
+
+When the whitelist is enabled, only whitelisted applications can be used by organization users. If a user attempts to authorize a non-whitelisted app, they will be prompted to contact their organization administrator.
+
+image::connected-apps-whitelist-blocked.png[The interface the user sees when an app is not on the whitelist, 40%,40%]
```
Comment (sarathecoubian) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
image::connected-apps-whitelist-blocked.png[The interface the user sees when an app is not on the whitelist, 50%,50%]
```

File: modules/ROOT/pages/connected-apps-org-admin.adoc (hunk 15 above)
Comment (maxwellgerber) [inline modules/ROOT/pages/connected-apps-org-admin.adoc]:
```suggestion
Previously whitelisted apps _must_ be whitelisted again with the new set of scopes before your users can grant those additional scopes to the app.
```

File: modules/ROOT/pages/connected-apps-developers.adoc (hunk 5 above)
Comment (maxwellgerber) [reply modules/ROOT/pages/connected-apps-developers.adoc]: I figured whitelisting is the action - the whitelist is the object that gets modified when the admin goes out and does some whitelisting

Comment (maxwellgerber) [reply modules/ROOT/pages/connected-apps-developers.adoc]:
```suggestion
== Application Whitelist
```

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**